Tests/UnitTests/Login/LoginCoordinatorTests.swift

```diff
@@ -152,11 +149,14 @@ extension LoginCoordinatorTests {
     func test_getAccessToken_errors() throws {
         mockDefaultStore.set(Date() + 60, forKey: .accessTokenExpiry)
         // GIVEN the secure store returns an error from reading an item
-        mockSecureStore.errorFromReadItem = SecureStoreError.generic
+        mockSecureStore.errorFromReadItem = SecureStoreError.unableToRetrieveFromUserDefaults
         // WHEN the LoginCoordinator's getAccessToken method is called
         sut.getAccessToken()
         // THEN the token holder's access token property should not get the access token from secure store
         XCTAssertEqual(sut.tokenHolder.accessToken, nil)
+        // THEN login flow should be triggered
+        XCTAssertTrue(sut.root.viewControllers.count == 1)
+        XCTAssertTrue(sut.root.topViewController is IntroViewController)
```

Comment:
I think you could probably add something to the mocks to validate the `refreshStorage()` method is being called 🙂